service_configuration_lib/spark_config.py

```diff
@@ -1057,6 +1057,10 @@ def get_spark_conf(
         spark_conf, cluster_manager, paasta_pool, force_spark_resource_configs,
     )
 
+    # Enabling auto-decommission feature to move data around makes sense for only spot nodes
+    if 'batch' != paasta_pool and 'spark.decommission.enabled' in spark_conf:
```

Comment:
this seems a tad brittle/hard to change - should this come from srv-configs/puppet instead?

additionally, wouldn't this disable the decommission feature for the spot gpu pools?

Reply:
we don't have spot gpu pools as yet and no plans as such in near future. 
I think having them here for now if fine.